korma/src/commonMain/kotlin/com/soywiz/korma/geom/Rectangle.kt

```diff
@@ -302,6 +303,11 @@ fun RectangleInt.setBoundsTo(left: Int, top: Int, right: Int, bottom: Int) = set
 ////////////////////
 
 operator fun IRectangleInt.contains(v: SizeInt): Boolean = (v.width <= width) && (v.height <= height)
+operator fun IRectangleInt.contains(that: IPoint) = contains(that.x, that.y)
+operator fun IRectangleInt.contains(that: IPointInt) = contains(that.x, that.y)
+fun IRectangleInt.contains(x: Double, y: Double) = (x >= left && x < right) && (y >= top && y < bottom)
+fun IRectangleInt.contains(x: Float, y: Float) = contains(x.toDouble(), y.toDouble())
+fun IRectangleInt.contains(x: Int, y: Int) = contains(x.toDouble(), y.toDouble())
```

Comment:
Dear Internet journeyman. If you are reading these lines, you can still understand the motivation of this, but still suffer as much as I do, and your retinas are shedding from your eyes in pain, and there are no onions near. Join my cause: https://youtrack.jetbrains.com/issue/KT-39136